web/yo/app/scripts/controllers/gene.js

```diff
@@ -1647,10 +1644,19 @@ angular.module('oncokbApp')
                 }
                 // review mode check
                 if (mutationReview && mutationReview.get('removed') || tumorReview && tumorReview.get('removed') || treatmentReview && treatmentReview.get('removed')) {
+                    if(reviewObj && !reviewObj.get('removedItem')) {
```

Comment:
That is a very good point. I just put up together [a document ](https://docs.google.com/document/d/1mNMKMOKvigYe9bRqjpmVmJebVbfy_yBus0igKhcj1xI/edit)for all the possible status in _review.